pkg/process/checks/process.go

```diff
@@ -678,3 +689,8 @@ func isDisallowListed(cmdline []string, disallowList []*regexp.Regexp) bool {
 	}
 	return false
 }
+
+func initZombieDisallowed(config ddconfig.Reader) bool {
+	disallowed := config.GetBool(configDisallowZombies)
+	return disallowed
+}
```

Comment:
[minor] This function doesn't really do anything besides make a call to `getBool`. You should just inline it.